pyccl/background.py

```diff
@@ -11,6 +11,23 @@
     'neutrinos_massive': lib.omega_nu_label,
 }
 
+species_types_rho = {
```

Comment:
Again, let's try to recycle 'species_types' here. I think you'll introduce a lot of extra complexity otherwise